src/main/java/org/snomed/snowstorm/core/data/services/BranchReviewService.java

```diff
@@ -468,11 +471,141 @@ private void lookupBranchReviewConceptChanges(BranchReview branchReview) {
 		start.setTime(start.getTime() + 1);
 
 		Set<Long> changedConcepts = createConceptChangeReportOnBranchForTimeRange(source.getPath(), start, source.getHead(), branchReview.isSourceParent());
+		changedConcepts.addAll(getContradictoryConceptIdentifiers(branchReview, source, target));
 		branchReview.setStatus(ReviewStatus.CURRENT);
 		branchReview.setChangedConcepts(changedConcepts);
 		branchReviewRepository.save(branchReview);
 	}
 
+	/*
+	 * Return Concept identifiers where one side of the merge has added a Description to a Concept, whilst the other
+	 * side has deleted the Concept entirely.
+	 * */
+	private Set<Long> getContradictoryConceptIdentifiers(BranchReview branchReview, Branch source, Branch target) {
+		Set<Long> contradictoryConceptIdentifiers = new HashSet<>();
+
+		// Deleted on Source but modified on Target
+		String sourcePath = source.getPath();
+		String targetPath = target.getPath();
+		Set<String> versionsReplaced = branchService.findLatest(targetPath).getVersionsReplaced(Concept.class);
+		if (!versionsReplaced.isEmpty()) {
+			List<Long> concepts = new ArrayList<>(mapInternalIdentifiersToConceptIdentifiers(versionsReplaced, sourcePath));
+			removeIfFoundOnPath(concepts, targetPath); // If found, then not deleted
+
+			contradictoryConceptIdentifiers.addAll(concepts);
+		}
+
+		// Modified on Source but deleted on Target
+		List<String> conceptIdsWithModifiedDescriptions = new ArrayList<>();
+		try (final SearchHitsIterator<Description> stream = elasticsearchTemplate.searchForStream(
+				newSearchQuery(versionControlHelper.getUpdatesOnBranchDuringRangeCriteria(sourcePath, getStart(branchReview, source, target), source.getHead())).build(), Description.class)
+		) {
+			stream.forEachRemaining(hit -> conceptIdsWithModifiedDescriptions.add(hit.getContent().getConceptId()));
+		}
+
+		Map<Long, Date> conceptsEndedOnTarget = new HashMap<>();
+		for (String conceptId : conceptIdsWithModifiedDescriptions) {
+			try (final SearchHitsIterator<Concept> stream = elasticsearchTemplate.searchForStream(new NativeSearchQueryBuilder()
+					.withSort(SortBuilders.fieldSort(Description.Fields.START).order(SortOrder.DESC))
+					.withQuery(
+							boolQuery()
+									.must(matchQuery(Description.Fields.CONCEPT_ID, conceptId))
+									.must(matchQuery(Description.Fields.PATH, targetPath))
+					)
+					.build(), Concept.class)) {
+				stream.forEachRemaining(hit -> {
+					Concept concept = hit.getContent();
+
+					boolean endedOnTarget = concept.getEnd() != null && targetPath.equals(concept.getPath());
+					if (endedOnTarget) {
```

Comment:
@dmcgihtsdo We can add existsQuery to check whether "end" field is present or not.

Reply:
Good thinking @CoderMChu ; I will add that in.